src/MainOpt.cpp

```diff
@@ -133,7 +133,7 @@ std::pair<int, std::unique_ptr<MainOpt>> parse_opt(int argc, char **argv) {
   App.add_option("--graylog-logger-address", opt.GraylogLoggerAddress,
                  "Address for Graylog logging");
   App.add_option("--influx-url", opt.InfluxURI, "Address for Influx logging");
-  App.add_option("-v,--verbose", log_level, "Syslog logging level", true)
+  App.add_option("-v,--verbosity", log_level, "Syslog logging level", true)
```

Comment:
Out of interest: What's the motivation for the change?

Reply:
Thanks! I just asked out of curiosity, I use anyway only the short version.

Reply:
me too, but for the INIs you have to put the long version in, i think.